package.json

```diff
@@ -106,14 +106,19 @@
     "angular-cookies": "^1.7.9",
     "angular-route": "^1.7.9",
     "bootstrap": "^3.4.1",
+    "create-react-class": "15.6.2",
     "cropperjs": "^1.5.6",
     "default-passive-events": "^1.0.10",
     "gif-frames": "^1.0.1",
     "gifshot": "^0.4.5",
+    "literallycanvas": "0.5.2",
     "lodash": "^4.17.14",
     "mathjs": "^6.6.4",
     "moment": "^2.24.0",
     "nerdamer": "^1.1.4",
+    "react": "16.1.1",
+    "react-dom": "16.1.1",
+    "react-dom-factories": "1.0.2",
```

Comment:
Why are these added?

Reply:
I think these are dependencies for literallyCanvas. @rohitkatlaa, is that correct?

Reply:
Yes these dependencies are necessary for literallyCanvas.

Reply:
Oof, what is their size, also why do we need to list them here and why aren't they imported by only including `literallyCanvas`?

Reply:
Literally Canvas  does not include react by its own. I have verified it by trying to include only literallycanvas and it gave an error stating the react can't be found. This error was resolved once I installed react.
React - 127KB
ReactDom - 1.3MB
React-dom-factories - 7.7KB
create-react-class - 82.1KB

Reply:
Are these sizes after the files are minified and gzipped, if not could you please provide these? Also, this is quite a lot for one library, are there any other alternatives?

Reply:
Here's what I found fo react and react-dom:
Package | Size (minified)
------------ | -------------
react | 6.5 kb ([source](https://unpkg.com/browse/react@16.1.1/umd/))
react-dom | 94 kb ([source](https://unpkg.com/browse/react-dom@16.1.1/umd/))

Not sure about the other packages. @rohitkatlaa can you check? Also, can you double-check if literallycanvas actually requires the other packages to work?

Also, among the suggested installation method for literallycanvas [here](https://github.com/literallycanvas/literallycanvas-demos), are we using the npm_webpack method? The description seems to fit our project.

@vojtechjelinek, per @rohitkatlaa's proposal there does not seem to be good alternatives. This package is simple, configurable, and has great browser support. React is the only dependency for this and I think it is reasonable (considering that the package sizes are quite small for the main packages react & react-dom -- I don't think the others are required but @rohitkatlaa  to confirm).

Reply:
I have checked and confirmed that LiterallyCanvas requires both create-react-class and react-dom-factories.  I am not able to find any data for minified and gzipped versions for these packages.

Reply:
Package | Size (minified)
-- | -- 
create-react-class | 7.19 kb ([source](https://unpkg.com/browse/create-react-class@15.6.2/))
react-dom-factories | 3 kb ([source](https://www.jsdelivr.com/package/npm/react-dom-factories))

Reply:
Based on the source you provided, these are the gzipped sizes.
![sizes_nobg](https://user-images.githubusercontent.com/11008603/82834540-11b21300-9edf-11ea-8fb9-b7a4cc7c5850.png)

(see "transferred over the network" values)

Can you add all this info in the description of the PR please?

Reply:
I have added all the information to the description of the PR.

Reply:
And thank you for explaining me about the gzipped files.